gddragonbones.cpp

```diff
@@ -723,7 +731,140 @@ void GDDragonBones::stop(bool _b_all)
     _reset();
 }
 
-float GDDragonBones::tell() const
+const DragonBonesData *GDDragonBones::get_dragonbones_data() const {
+	const DragonBonesData* data = this->p_armature->getArmature()->getArmatureData()->getParent();
+	return data;
+}
+
+ArmatureData* GDDragonBones::get_armature_data(const String &_armature_name) {
+	std::map<std::string, ArmatureData *>::const_iterator data = get_dragonbones_data()->armatures.find(_armature_name.ascii().get_data());
+
+	if (data == get_dragonbones_data()->armatures.end()) {
+		return nullptr;
+	}
+
+	return data->second;
+}
+
+GDArmatureDisplay *GDDragonBones::get_armature_display(const String &_armature_name) {
+	auto *armature = get_armature_data(_armature_name);
```

Comment:
This is a flawed way of doing this; but it was a good way to flesh out the structure. On loading, probably in `set_parent_class` call where I loop over all the slots; I'll build a slot->childarmaturedisplays cache in order to do easy referencing at the top level